core/src/main/java/com/scalar/db/transaction/consensuscommit/CommitHandler.java

```diff
@@ -397,7 +423,7 @@ public void rollbackRecords(Snapshot snapshot) {
       }
       parallelExecutor.rollbackRecords(tasks, snapshot.getId());
     } catch (Exception e) {
-      logger.warn("Rolling back records failed. Transaction ID: {}", snapshot.getId(), e);
+      logger.info("Rolling back records failed. Transaction ID: {}", snapshot.getId(), e);
```

Comment:
Diito. Changed log level from `warn` to `info` because it is a normal occurrence and failing to roll back records does not affect correctness.